runner/ansible/vars/azure_env.yml

```diff
@@ -17,3 +17,10 @@ expected:
   "1.3.4": "3"
   "1.3.5": "60"
   "1.3.6": "120"
+  "1.4.1.monitor_interval": "3600"
+  "1.4.1.monitor_timeout": "120"
+  "1.4.1.pcmk_action_limit": "3"
+  "1.4.1.pcmk_monitor_retries": "4"
+  "1.4.1.pcmk_reboot_timeout": "900"
+  "1.4.1.power_timeout": "240"
+  "1.4.2": "true"
```

Comment:
We can remove this `1.4.2`. It is not used

Reply:
No, it is used now:
    [[ $(crm_attribute -t crm_config -G -n concurrent-fencing --quiet) = "{{ expected[test_id] }}" && exit 0